### PR TITLE
Move track order

### PR DESCRIPTION
**Pull request by binalee21**

### CODE REVIEW

File: cypress/integration/moveTrackOrder.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            });       
+    });
+
+    it('move the track & verify new order', () => {       
+        if(startRow < destinationRow) {
+            //custom command to move tracks down
+            cy.moveTrackDown(startRow, destinationRow, trackTitle);
+
+            //verify moved track title
+            cy.get('playlist-tracks')
```
Comment (Shreya76) [inline cypress/integration/moveTrackOrder.js:37]: We can keep this verification outside the if else block, so we don't have to repeat the code.

**Review by Shreya76 — CHANGES_REQUESTED** [review]

**Comment by binalee21** [comment]
changes done - verification code outside the if block.

File: cypress/integration/moveTrackOrder.js (hunk 1 above)
Comment (binalee21) [reply cypress/integration/moveTrackOrder.js]: changes done - verification code outside the if block.

**Comment by binalee21** [comment]
> @binalee21 Please resolve conflicts

Done

**Review by Shreya76 — APPROVED** [review]

**Pull request merged**